project/SbtMultiJvm.scala

```diff
@@ -372,11 +374,41 @@ object MultiJvmPlugin extends AutoPlugin {
         val connectInput = input && index == 0
         log.debug("Starting %s for %s".format(jvmName, testClass))
         log.debug("  with JVM options: %s".format(allJvmOptions.mkString(" ")))
-        (testClass, Jvm.startJvm(javaBin, allJvmOptions, runOptions, jvmLogger, connectInput))
+        val testClass2Process = (testClass, Jvm.startJvm(javaBin, allJvmOptions, runOptions, jvmLogger, connectInput))
+        if (index == 0) {
+          log.debug("%s for %s 's started as `Controller`, waiting before can be connected for clients.".format(jvmName,
+            testClass))
+          val controllerHost = hosts.head
+          val serverPort: Int = Integer.getInteger("multinode.server-port", 4711)
+          waitingBeforeConnectable(controllerHost, serverPort, TimeUnit.SECONDS.toMillis(20L))
+        }
+        testClass2Process
     }
     processExitCodes(name, processes, log)
   }
 
+  private def waitingBeforeConnectable(host: String, port: Int, timeoutInMillis: Long): Unit = {
+    val inetSocketAddress = new InetSocketAddress(host, port)
+    def telnet(addr: InetSocketAddress, timeout: Int): Boolean = {
+      val socket: Socket = new Socket()
+      try {
+        socket.connect(inetSocketAddress, timeout)
+        socket.isConnected
+      } catch {
+        case _: Exception => false
+      } finally {
+        socket.close()
+      }
+    }
+
+    val startTime = System.currentTimeMillis()
+    var connectivity = false
+    while (!connectivity && (System.currentTimeMillis() - startTime < timeoutInMillis)) {
```

Comment:
Same with previous comment about constants.

Reply:
I will resolve this for now, I don't want to block the PR because of this and if people have issues we can always update this later (its just tests)

Reply:
I think that's ok, as it's just in tests and no one will touch those again, otherwise we need another：
```
multinode.max-time-waiting-to-connect-in-seconds=20 
```
like thing